### PR TITLE
[v2.0.3] [Consent Module] Fixed Issue #3124

### PR DESCRIPTION
**Pull request by navya-BTC**
Fixed issue #3124: Removed Spring Security dependency and it's configuration as we are using Hydra for resource access

### CODE REVIEW

File: participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/MyStudiesUserRegWsApplication.java [hunk 1]
```diff
@@ -13,7 +13,9 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.scheduling.annotation.EnableScheduling;
```
Comment (moschetti) [inline participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/MyStudiesUserRegWsApplication.java:16]: Please update file copyright to `2020-2021`

File: participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java [hunk 2]
```diff
@@ -8,7 +8,22 @@
 
 package com.google.cloud.healthcare.fdamystudies.config;
 
+import com.google.cloud.storage.Storage;
```
Comment (moschetti) [inline participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java:11]: Please update file copyright to `2020-2021`

File: participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/MyStudiesUserRegWsApplication.java (hunk 1 above)
Comment (navya-BTC) [reply participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/MyStudiesUserRegWsApplication.java:16]: Fixed review comment.

File: participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java (hunk 2 above)
Comment (navya-BTC) [reply participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java:11]: Fixed review comment.

**Review by moschetti — APPROVED** [review]

**Pull request merged**